luaui/Widgets/gui_gridmenu.lua

```diff
@@ -928,10 +939,7 @@ function widget:Update(dt)
 	end
 
 	if selectNextFrame and not isPregame then
-		local cmdIndex = spGetCmdDescIndex(selectNextFrame)
-		if cmdIndex then
-			Spring.SetActiveCommand(cmdIndex, 1, true, false, Spring.GetModKeyState())
```

Comment:
Same here the arguments to `SetActiveCommand` are not the same, for reference: `Spring.SetActiveCommand(cmd, 1, false, true,Spring.GetModKeyState())`